packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixes avoid_redundant_argument_values lint warnings and minor typos.
 * Ignores unnecessary import warnings in preparation for [upcoming Flutter changes](https://github.com/flutter/flutter/pull/104231).
 * Updates references to the obsolete master branch.
+* Fix typo from lowercase to uppercase
```

Comment:
This needs to follow the changelog style guide linked from the PR description.

Reply:
done